### ISSUE TITLE
sort_intersections_by degree than by cardinality

### ISSUE THREAD

**Issue by dariober** [krassowski/complex-upset#47]
Hi- I just started playing with ComplexUpset (0.5.15), thank you - great stuff!

As per title, it would be useful to have the option to have columns in the intersection matrix ordered by degree than by cardinality. E.g. like `sort_intersections_by = c('degree', 'cardinality')`. 

----

Probably a separate feature request: It would be nice if the `intersect` parameter (or some other parameter) could take a list of lists of intersections and plot only those (similar to the original upset in UpSetR), optionally respecting the order in the outer list

**Comment by krassowski** [comment]
Both ideas sound great!

**Comment by krassowski** [comment]
Thank you for the suggestion! Sorting by multiple criteria is now available as a part of 0.8.0 development release. To get it the newest version please use:

```R
if(!require(devtools)) install.packages("devtools")
devtools::install_github("krassowski/complex-upset")
```

I moved the other suggestion to a separate issue (#75) - you can subscribe to updates by clicking the bell icon in the other thread. Thanks again!
